include/libopencm3/stm32/g0/dmamux.h

```diff
@@ -0,0 +1,175 @@
+/** @defgroup dmamux_defines DMAMUX Defines
+ *
+ * @ingroup STM32G0xx_defines
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2019
+ * Guillaume Revaillot <g.revaillot@gmail.com>
+ *
+ * @brief Defined Constants and Types for the STM32G0xx DMAMUX DMA request router
+ *
+ * @version 1.0.0
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_DMAMUX_H
+#define LIBOPENCM3_DMAMUX_H
+/**@{*/
+
+#include <libopencm3/stm32/common/dmamux_common_all.h>
+
+ /** @defgroup dmamux_reg_base DMAMUX register base addresses
+  * @{
+  */
+#define DMAMUX1				DMAMUX_BASE
+/**@}*/
+
+/* --- DMAMUX_CxCR values ------------------------------------ */
+
+/** @defgroup dmamux_cxcr_sync_id SYNCID Synchronization input selected
+  @ingroup dmamux_defines
```

Comment:
fyi, you don't need @ingroup markings unless you're trying to put them somewhere else than their logical location.  You're already in dmamux_defines from the defgroup on line 1.

Reply:
indeed - and you pointed that already in more recent PRs i think ... that bit has been on my repo for too long :)